VEHICLE/IsVehicleStoppedAtTrafficLights.md

```diff
@@ -9,10 +9,10 @@ BOOL IS_VEHICLE_STOPPED_AT_TRAFFIC_LIGHTS(Vehicle vehicle);
 ```
 
 ```
-Is this for red lights only?  more testing required.
+This native checks if the given vehicle is stopped at a red light.
+
+**Note:** Will return false if the vehicle is stopped behind another vehicle waiting at a traffic light.
```

Comment:
false -> `false`

Reply:
This won't work since it's already in a \`\`\` box no ?

Reply:
Imported comments shouldn't be modified, instead the fenced code block should be removed and documentation should be written instead, as a description of the native. That way everyone knows the documentation has been updated, it will show "Has modern docs" in the native reference site.